pkg/controllers/cloud/configmap_controller.go

```diff
@@ -30,13 +30,41 @@ import (
 	"antrea.io/nephe/pkg/controllers/config"
 )
 
+type ConfigMapHandler func(controllerConfig *config.ControllerConfig)
+
+type controllerConfigType int
+
 type ConfigMapReconciler struct {
 	client.Client
 	Log              logr.Logger
 	Scheme           *runtime.Scheme
 	ControllerConfig *config.ControllerConfig
 }
 
+const (
+	controllerConfigTypeInterval controllerConfigType = iota
+)
+
+var (
+	configmapHandlerFunc map[controllerConfigType][]ConfigMapHandler
```

Comment:
move this inside ConfigMapReconciler type

Reply:
Solved.